testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -32,6 +32,7 @@
     <description></description>
 
     <properties>
+        <browser>phantomjs</browser>
```

Comment:
I think this is not good as it changes the default browser to phantomJS for the whole testsuite. I believe that's the reason why there are travis failures. It is possible to "hardcode" the default browser just for the WelcomePageTest with the stuff that @slaskawi recently added for the X509 tests. See for example X509BrowserLoginTest and this PR https://github.com/keycloak/keycloak/pull/5930

Reply:
fixed